.cloudcannon/routing.json

```diff
@@ -1066,7 +1066,7 @@
     },
     {
       "from": "/documentation/articles/configuring-toolbars/",
-      "to": "/documentation/articles/configure-your-wysiwyg-toolbars/",
+      "to": "/documentation/articles/configure-your-rich-text-editors/",
```

Comment:
@EllaCloudCannon since [configure-your-wysiwyg-toolbars](https://cloudcannon.com/documentation/articles/configure-your-wysiwyg-toolbars/) is currently a linkable URL, we'll probably want another redirect from that to the new page.